src/anomalib/pipelines/benchmark/generator.py

```diff
@@ -15,6 +15,19 @@
 from .job import BenchmarkJob
 
 
+def flatten_dict(d: dict | object, key: str = "", output: dict | None = None) -> dict:
```

Comment:
Can we not use this method? https://github.com/openvinotoolkit/anomalib/blob/3a403ae8c6cf91d9fb52aa58a198d559a71dfd00/src/anomalib/utils/config.py#L143

Reply:
Sure. I was not aware of the existing function. Fixed in https://github.com/openvinotoolkit/anomalib/pull/2391/commits/d7ecb0a990ba5f1761606178df2f98cf4ef30a1a